xml/favicons.xml

```diff
@@ -423,7 +423,7 @@
     <example>5856edf7bcbea0817312d9e535e5eb2a</example>
     <example>f4f3cb900258441d5dbc9105b7ab9b44</example>
     <example>c6acedaff906029fc5455d9ec52c7f42</example>
-    <param pos="0" name="service.vendor" value="VMWare"/>
+    <param pos="0" name="service.vendor" value="VMware"/>
```

Comment:
Here and elsewhere: Correcting the case of `VMware`.

Reply:
Verified this is how we match in product.

Reply:
Thanks. It was mixed use (`VMWare` vs `VMware`) throughout the fingerprints but, IIRC, most of our fingerprints were using the correct case.  This is a good example of  why this tool has value.